packages/extension/.env

```diff
@@ -1,11 +1,11 @@
-NEXT_PUBLIC_API_URL=http://api.local.com
+NEXT_PUBLIC_API_URL=https://api.local.fylla.dev
 #NEXT_PUBLIC_API_URL=https://api.daily.dev
-NEXT_PUBLIC_SUBS_URL=ws://api.local.com/graphql
-NEXT_PUBLIC_DOMAIN=local.com
+NEXT_PUBLIC_SUBS_URL=ws://api.local.fylla.com/graphql
+NEXT_PUBLIC_DOMAIN=fylla.com
```

Comment:
```suggestion
NEXT_PUBLIC_SUBS_URL=ws://api.local.fylla.dev/graphql
NEXT_PUBLIC_DOMAIN=local.fylla.dev
```

Reply:
For some weird reasons, GH is not accepting the suggestion 🤔 

I'll update them manually.

![Screenshot 2025-02-24 at 9 51 22 PM](https://github.com/user-attachments/assets/698564e7-da5a-47af-87c6-0d28f4c465eb)